src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+  const propertyValues = Object.values(robot);
```

Comment:
The variable 'propertyValues' is not necessary since you can check for duplicate values and construct the inverse object in a single loop.